lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -338,6 +339,10 @@ export class StandardController implements IController {
         );
         this.eventHandler.emitEvent(EventType.INITIALIZE_START);
 
+        if (!request?.isBroker) {
+            this.logMultipleInstances(initMeasurement);
```

Comment:
Let's wrap this in a try/catch - I don't want unexpected failures here to result in being unable to use the lib at all